.gitignore

```diff
@@ -286,3 +286,10 @@ tmp-php.ini
 !/ext/fileinfo/magicdata.patch
 !/ext/pcre/pcre2lib/config.h
 !/win32/build/Makefile
+
+# -------
```

Comment:
https://stackoverflow.com/a/7335487/1320374 :wink:

Reply:
Thanks @iluuu1994. I was going to take care of this before I opened the PR from draft.

Reply:
Np, I thought maybe you didn't know about global gitignores.

Reply:
I actually didn't, so the comment was definitely helpful. :)